src/core/featuremodel.h

```diff
@@ -51,7 +54,8 @@ class FeatureModel : public QAbstractListModel
       AttributeName = Qt::UserRole + 1,
       AttributeValue,
       Field,
-      RememberAttribute
+      RememberAttribute,
+      LinkedAttribute
```

Comment:
Can you add a docstring about what a "LinkedAttribute" is? (`//!< docstring goes here`)